R/0_documentation.R

```diff
@@ -0,0 +1,33 @@
+rd_disease_threshold <- function(type = "param", usage = NULL) {
+  checkmate::assert_choice(type, "param")
```

Comment:
The `type = "param"` argument was something I needed for R6 classes. I don't think you need it for your work

```suggestion
rd_disease_threshold <- function(usage = NULL) {
```